lib/QuickNavigation/Minibar/ArticleHistoryElement.php

```diff
@@ -11,7 +11,7 @@
 
 namespace FriendsOfRedaxo\QuickNavigation\Minibar;
 
-use FriendsOfRedaxo\QuickNavigation\Buttons\ArticleHistory;
+use FriendsOfRedaxo\QuickNavigation\Button\ArticleHistory;
```

Comment:
```suggestion
use FriendsOfRedaxo\QuickNavigation\Button\ArticleHistoryButton;
```

weiter unten müsstest du den Aufruf anpassen.